src/components/CommentList/CommentList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
-import './CommentList.scss';
+import { ComObj } from '../../type';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList: React.FC <ComObj> = (
+  {
+    comms,
+  },
+) => (
+  <ul>
+    {comms.map(com => (
+      <li key={com.id}>
+        <CommentInfo {...com} />
```

Comment:
CommentInfo component should accept full comment object
<CommentInfo comment={...} />